From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-hugetlb-enforce-brace-style-checkpatch-fixes
Date: Thu Jan 15 11:25:21 AM PST 2026

WARNING: suspect code indent for conditional statements (16, 25)
#63: FILE: mm/hugetlb.c:2678:
+		} else {
[...]
 			 vma_end_reservation(h, vma, address);

total: 0 errors, 1 warnings, 58 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

./patches/mm-hugetlb-enforce-brace-style.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: David Hildenbrand <david@kernel.org>
Cc: Joshua Hahn <joshua.hahnjy@gmail.com>
Cc: Muchun Song <muchun.song@linux.dev>
Cc: Oscar Salvador <osalvador@suse.de>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/hugetlb.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/mm/hugetlb.c~mm-hugetlb-enforce-brace-style-checkpatch-fixes
+++ a/mm/hugetlb.c
@@ -2679,8 +2679,7 @@ void restore_reserve_on_error(struct hst
 			/*
 			 * No reservation present, do nothing
 			 */
-			 vma_end_reservation(h, vma, address);
-
+			vma_end_reservation(h, vma, address);
 		}
 	}
 }
_
